libvirt/domain.go

```diff
@@ -280,17 +281,117 @@ func inlineUpdateCoreOsIgnition(d *schema.ResourceData, domainDef *libvirtxml.Do
 		if err != nil {
 			return err
 		}
-		err = guestfishExecution(rootPath, fileToUpload)
+		connURI, err := virConn.GetURI()
+		if err != nil {
+			return err
+		}
+		err = guestfishExecution(rootPath, fileToUpload, connURI)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
+func getLibvirtHostFromURI(connURI string) string {
+	// example uri with all feilds: schema://userinfo@host:port/system
```

Comment:
I'd instead provide [this link](https://libvirt.org/uri.html) here.

Reply:
Agreed